### DISCUSSION TITLE
FRT PORTAL

### DISCUSSION THREAD

**Discussion by 19C91A0503** [github/india#409] [Students]
@ey-mentor1 @ey-mentor2 , @EY-Mentor3
Respected sir/madam,
   my name is siddhartha. I completed watching all the industry session videos. even though it is showing as incomplete. I didn't skip any video. I also tried it in incognito. I also tried it by clearing cache and restarting website. I also tried in different browsers.

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any `Official FRT Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Currently, we are helping all the `Future Ready Talent` Interns clear their doubts & queries on a personal basis either on chat or voice, whichever they prefer. We will also be assisting them on Projects & help them complete this internship 🤗

Link to join: https://discord.gg/drQ7ySCdX2
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏

**Reply by PRIYA-S-102** [reply]
have you completed your FRT Internship project if yes help me for completing.
mail me at princessrajpriya26@gmail.com

**Comment by TheSuranaverse** [comment]
@19C91A0503 join the community & connect with me there, we have solved more than 250+ students doubts regarding that

**Reply by PRIYA-S-102** [reply]
have you completed your FRT Internship project if yes help me for completing.
mail me at princessrajpriya26@gmail.com
